src/style-placeholder.js

```diff
@@ -13,8 +13,25 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
 import {applyStylePlaceHolder} from './style-util.js';
 import {nativeShadow} from './style-settings.js';
 
-/** @type {Object<string, !Node>} */
-let placeholderMap = {};
+/** @type {!Object<string, !Node>} */
+const placeholderMap = {};
+
+/**
+ * @param {string} elementName
+ * @return {Node}
+ */
+export function getStylePlaceholder(elementName) {
+  return placeholderMap[elementName] || null;
+}
+
+/**
+ * @param {string} elementName
+ */
+export function addStylePlaceholder(elementName) {
```

Comment:
Maybe change name to `ensureStylePlaceholder` since it only makes one if it doesn't exist.

Reply:
Ok